src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -109,6 +109,7 @@
 import org.opensearch.http.HttpServerTransport;
 import org.opensearch.http.HttpServerTransport.Dispatcher;
 import org.opensearch.http.netty4.ssl.SecureNetty4HttpServerTransport;
+import org.opensearch.identity.PluginSubject;
```

Comment:
I think the rebase picked changes from a different commit?

Reply:
Rebased. I pulled in these changes from https://github.com/opensearch-project/security/pull/4690 to fix compilation errors after https://github.com/opensearch-project/OpenSearch/pull/14630 was merged into core.